cmd/influx/authorization.go

```diff
@@ -77,6 +78,10 @@ func authCreateCmd() *cobra.Command {
 
 	cmd.Flags().StringVarP(&authCreateFlags.org, "org", "o", "", "The organization name (required)")
 	cmd.MarkFlagRequired("org")
+	viper.BindEnv("ORG")
+	if h := viper.GetString("ORG"); h != "" {
+		authCreateFlags.org = h
```

Comment:
this feels off to me, I would suggest flags take precendence over env vars personally